cookbooks/bcpc/attributes/default.rb

```diff
@@ -8,17 +8,12 @@
 default['bcpc']['location'] = 'New York'
 default['bcpc']['organization'] = 'Bloomberg'
 
-# Should be kvm (or qemu if testing in VMs)
-default['bcpc']['virt_type'] = 'kvm'
-
 # Region name for this cluster
 default['bcpc']['region_name'] = node.chef_environment
 
 # Domain name that will be used for DNS
 default['bcpc']['domain_name'] = 'bcpc.example.com'
```

Comment:
Good call; I've cleaned that up and a few other duplicated and unused attributes.